common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -862,24 +873,33 @@ def _parse_video_xml(cls, xml, id_generator=None):
         if 'download_track' not in field_data and track is not None:
             field_data['download_track'] = True
 
-        video_asset_elem = xml.find('video_asset')
-        if (
-                edxval_api and
-                video_asset_elem is not None and
-                'edx_video_id' in field_data
-        ):
-            # Allow ValCannotCreateError to escape
-            edxval_api.import_from_xml(
-                video_asset_elem,
-                field_data['edx_video_id'],
-                course_id=course_id
-            )
-
         # load license if it exists
         field_data = LicenseMixin.parse_license_from_xml(field_data, xml)
 
         return field_data
 
+    def import_video_info_into_val(self, xml, course_id):
+        """
+        Import parsed video info from `xml` into edxval.
+
+        Arguments:
+            xml (lxml object): xml representation of video to be imported
+            course_id (str): course id
+        """
+        clean = lambda item: item.strip() if isinstance(item, basestring) else item
```

Comment:
We should not assign lambda expressions. Assigning lambdas to names basically just duplicates the functionality of `def`: https://www.python.org/dev/peps/pep-0008/#programming-recommendations

Reply:
```
if self.edx_video_id is not None:
     self.edx_video_id = self.edx_video_id.strip()
````

Reply:
no need to compute `external`, decision can be made with `self.edx_video_id`